docs/RC/GetCapabilities/assets/GetCapabilities.txt

```diff
@@ -0,0 +1,44 @@
+participant App
+participant SDL
+participant HMI
+
+HMI->SDL:BC.OnReady
+SDL->HMI:RC.IsReady
+HMI-->SDL:RC.IsReady (code:0)
+note over SDL:<align:center>`HMICapabilitiesCacheFile`\nis not available
+SDL->SDL:Create `HMICapabilitiesCacheFile`
+
+opt Use of cached capabilities data
+
+opt On system start up
+note over SDL:<align:center> Capabilities data is not available\nin`HMICapabilitiesCacheFile`
+SDL->HMI:RC.GetCapabilities
+HMI-->SDL:RC.GetCapabilities (params)
+SDL->SDL:Cache received data\nin `HMICapabilitiesCacheFile`
+App->SDL:GetCapabilities
+SDL->SDL:Load cached data\nfrom **`HMICapabilitiesCacheFile`**
+SDL-->App:   GetCapabilities (params)
+end
+
+opt In subsequent ignition cycles
+note over SDL:<align:center> Capabilities data is available\nin`HMICapabilitiesCacheFile`
+App->SDL:GetCapabilities
+SDL->SDL:Load cached data\nfrom **`HMICapabilitiesCacheFile`**
+SDL-->App:      GetCapabilities (params)
+end
+end
+
+opt Use of default capabilities data - No response from HMI
+
+note over SDL:<align:center> Capabilities data is not available\nin`HMICapabilitiesCacheFile`
+SDL->HMI:RC.GetCapabilities
+note over HMI:<align:center>No response
+App->SDL:GetCapabilities
+SDL->SDL:Load default data\nfrom **`hmi_capabilities.json`**
+SDL-->App:       GetCapabilities (params)
+note over SDL:<align:center> Requests\ncapabilities data from HMI\nin subsequent ignition cycle
```

Comment:
@Mariia-Novosiadla Why do we miss 
_Use of default capabilities data - Some capabilities data is missing in HMI response_ case here? 
According to the [HMI API](https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/interfaces/HMI_API.xml#L6871) HMI can send `RC.GetCapabilities` response without `seatLocationCapability` for example

Reply:
added in 2c17b89d8414132ce04dac594def797312fc3ad2